### PR TITLE
Mention that Dask is open source in landing page

### PR DESCRIPTION
**Pull request by mrocklin**
We apparently forgot to mention this except in the header.
This integrates this message into a broader message about being part
of the PyData ecosystem.

cc @jorisvandenbossche @pitrou who noticed this last night.  If there is a way to put this more prominently without disrupting messaging I'm open to suggestions.

### CODE REVIEW

**Comment by pitrou** [comment]
This looks fine on the principle. Let's see how it looks like on the landing page once this is pushed :-)

**Comment by mrocklin** [comment]
image

**Comment by pitrou** [comment]
I think the first sentence """Dask grew out of the open source PyData ecosystem""" is superfluous.
So I would rephrase : """Dask is open source and freely available. It is developed in coordination with other community projects like Numpy, Pandas, and Scikit-Learn."""

**Comment by mrocklin** [comment]
Yes.  That sounds much better.  Thanks @pitrou .  Pushed.

**Comment by jorisvandenbossche** [comment]
Looks good (still not really prominent (I also don't know to what extent this is needed), but at least it is there)
